### ISSUE TITLE
 Version 1.1.74 broke 'export default' declarations

### ISSUE THREAD

**Issue by alejandroclaro** [wessberg/rollup-plugin-ts#48]
Changes from version 1.1.73 and 1.1.74 break 'export default' declaration. For example:

```typescript
// module.ts
export default class DefaultClass {
  private static readonly constant = 0;

  public static staticMethod(): void {
  	console.log(DefaultClass.constant)
  }
}
```

 in version 1.1.74 produces this declaration file (.d.ts, which is incorrect):

```typescript
declare class DefaultClass {
    private static readonly constant;
    static staticMethod(): void;
}
export { DefaultClass };
```

but it was correct in version 1.1.73 (.d.ts):

```typescript
export default class DefaultClass {
    private static readonly constant;
    static staticMethod(): void;
}
```

I tested it with this configuration (rollup.config.js):

```javascript
const Path = require('path');

const alias = require('@rollup/plugin-alias');
const builtins = require('builtin-modules');
const commonjs = require('rollup-plugin-commonjs');
const filesize = require('rollup-plugin-filesize');
const json = require('@rollup/plugin-json');
const postcss = require('rollup-plugin-postcss');
const resolve = require('rollup-plugin-node-resolve');
const typescript = require('@wessberg/rollup-plugin-ts');
const vue = require('rollup-plugin-vue');

const config = {
  treeshake: false,
  plugins: [
    alias({
      'resolve': [ '.js', '.ts', '' ],
      'entries': [
        { 'find': '@src', 'replacement': Path.join(__dirname, 'src') },
      ]
    }),
    resolve(),
    commonjs(),
    json({ 'compact': true }),
    filesize(),
    typescript({
      'tsconfig': (resolvedConfig) => {
        const config = Object.assign({}, resolvedConfig);
        config.declarationDir = Path.join(__dirname, 'dist', 'types');
        return config;
      }
    }),
    vue(),
    postcss()
  ],
  external: [
  	...builtins
  ],
  input: './src/module.ts',
  output: [
    { file: 'dist/rollup-plugin-ts-bug.cjs.js', format: 'cjs', exports: 'named', sourcemap: true } 
  ]
}

export default config;
```

**Comment by wessberg** [comment]
Hi there. Sorry about that. It has been fixed in [v1.1.77](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.1.77)

**Comment by alejandroclaro** [comment]
Thank you @wessberg !
